monsteer/plugin/endOfStream.fbs

```diff
@@ -6,7 +6,5 @@ namespace monsteer.plugin;
 
 table EndOfStream
 {
-endofstream:bool = true;
+  endofstream:bool = true;
```

Comment:
This is actually an empty event. The attribute was added a temporary workaround for flat buffer event with not payload not being properly supported.
I told @bilgili to add a comment so we wouldn't forget about removing the attribute, but he was too sure (or maybe eager?) that he would do the change.